### PR TITLE
Observation normalizer for subset of fields

### PR DESCRIPTION
**Pull request by emailweixu**
Also added summary for observation

### CODE REVIEW

File: alf/nest/nest.py [hunk 1]
```diff
@@ -691,3 +691,39 @@ def _traverse(nested, levels):
...
+            return nested._replace(**{level: new_val})
+        elif isinstance(nested, dict):
+            new_val = nested.copy()
+            new_val[level] = _traverse(nested=nested[level], levels=levels[1:])
+            return new_val
+        else:
+            raise TypeError("If value is a nest, it must be either " +
+                            "a dict or namedtuple!")
+
+    return _traverse(nested=nested, levels=field.split('.') if field else [])
```
Comment (le-horizon) [inline alf/nest/nest.py]: Can this be done using transform_nest, e.g.
`result = alf.nest.utils.transform_nest(result, field, lambda _: new_value)`?

**Review by le-horizon — COMMENTED** [review]
Almost good, only one comment.

File: alf/nest/nest.py (hunk 1 above)
Comment (emailweixu) [reply alf/nest/nest.py]: Sure it can. But not feeling well to use more complex API to do the simpler task.

Comment (hnyu) [reply alf/nest/nest.py]: Right. But I think that to reduce maintenance cost, this can just be a wrapper instead that calls transform_nest

Comment (le-horizon) [reply alf/nest/nest.py]: If there isn't much efficiency gain, then I'd vote for less code.  Having set_field func call transform_nest is probably a good compromise.

File: alf/nest/nest.py [hunk 2]
```diff
@@ -693,37 +693,73 @@ def _traverse(nested, levels):
     return _traverse(nested=nested, levels=field.split('.') if field else [])
 
 
-def set_field(nested, field, new_value):
-    """Set the field in nested to ``new_value``.
+def transform_nest(nested, field, func):
```
Comment (emailweixu) [inline alf/nest/nest.py:696]: This is moved from nest/utils.py without any change.

**Review by le-horizon — DISMISSED** [review]
LG.  I'm surprised there are only so few calls to transform_nest.  I would imagine more places need to be updated.

**Review by hnyu — APPROVED** [review]

**Pull request merged**